frontend/src/components/movies/MovieDetails.js

```diff
@@ -8,271 +8,222 @@ import { popupNotification } from '../../lib/notification'
-          }  
-          <div className="comments-right">
-            <div className="comments-box">
-              <div className="comments-title">
-                <h1>Reviews</h1>
+        </div>
+      </div>
+      {/* Add review section START */}
+      <div className="next-page">
+        {isAuthenticated() && <div className="comments-left">
+          <form onSubmit={handleSubmit} className="review-column is-two-thirds is-offset-one-quarter box">
+            <div className="field">
+              <label className="label">Add A Review</label>
+              <div className="control">
+                <textarea
+                  className="textarea"
+                  placeholder="comments"
+                  name="text"
+                  value={text}
+                  onChange={handleChange}
+                />
               </div>
-              <hr />
-              <div className="user-reviews-parent">
-                <div className="user-reviews"> 
-                  {movie.comments.map((comment, id) => (
-                    <>
-                      <div key={id} className="comment-text">
-                        <h2>A review by {comment.user.username}</h2>
-                        <h3>Written on: {moment(comment.created_at).format('Do MMM YY')}</h3>
-                        <div className="comment-rating">
-                          <p><span>{comment.rating}.0</span></p>
-                        </div>
-                        <p>{comment.text}</p>
-                      </div>                    
+            </div>
+            <div className="field">
+              <label className="label">Rating</label>
+              <div className="control">
+                <input
+                  className="input"
+                  placeholder="Choose 1-5"
+                  name="rating"
+                  value={rating}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+            <div className="field">
+              <button type="submit" className="button is-fullwidth">Submit</button>
+            </div>
+          </form>
+        </div>}
+        {/* Add review section END */}
+        {!isAuthenticated() && <div className="login-to-review">
+          <h1>Login to rate and review movies</h1>
+          <Link to={'/login'} className="button">Login</Link>
+        </div>
+        }
+        <div className="comments-right">
+          <div className="comments-box">
+            <div className="comments-title">
+              <h1>Reviews</h1>
+            </div>
+            <hr />
+            <div className="user-reviews-parent">
+              <div className="user-reviews">
+                {movie.comments.map((comment, id) => (
+                  <>
+                    <div key={id} className="comment-text">
+                      <h2>A review by {comment.user.username}</h2>
+                      <h3>Written on: {moment(comment.created_at).format('Do MMM YY')}</h3>
```

Comment:
With your implementation of moment, try and create a separate component called `DateFormat` or something ... then you would pass it through:

```
<h3>Written on: <DateFormat date={comment.created_at} /></h3>
```

Then all of your date formatting is in one place.